sections/header.liquid

```diff
@@ -376,7 +379,6 @@
           <a href="{{ routes.root_url }}" class="header__heading-link link link--text focus-inset">
             {%- if section.settings.logo != blank -%}
               {%- assign logo_alt = section.settings.logo.alt | default: shop.name | escape -%}
-              {%- assign logo_height = section.settings.logo_width | divided_by: section.settings.logo.aspect_ratio -%}
```

Comment:
Also if we make the change here, we should probably update the password page too: https://github.com/Shopify/dawn/blob/main/sections/main-password-header.liquid#L3